modules/tinymce/src/themes/silver/main/ts/ui/general/Notification.ts

```diff
@@ -144,24 +144,14 @@ const factory: UiSketcher.SingleSketchFactory<NotificationSketchDetail, Notifica
     detail.level.bind((level) => Option.from(notificationIconMap[level])).toArray()
   ]);
 
-  return {
-    uid: detail.uid,
-    dom: {
-      tag: 'div',
-      attributes: {
-        role: 'alert'
-      },
-      classes: detail.level.map((level) => [ 'tox-notification', 'tox-notification--in', `tox-notification--${level}` ]).getOr(
-        [ 'tox-notification', 'tox-notification--in' ]
-      )
-    },
-    components: [ {
+  const components: AlloySpec[] = [
```

Comment:
I should add I pulled this out to remove the assertions done previously, which would have meant the specs weren't being typed checked.